scripts/tasks/src/swc.ts

```diff
@@ -43,14 +43,19 @@ async function swcTransform(options: Options) {
       outputPath,
     });
 
+    // Strip @jsx comments, see https://github.com/microsoft/fluentui/issues/29126
```

Comment:
heads up that using `@jsx` in a comment without it actually *being* a jsx pragma directive is causing eslint-plugin-react to crash, which is something we need to fix. See https://github.com/jsx-eslint/eslint-plugin-react/issues/3632.

Feel free in the meantime to avoid having `@` and `jsx` next to each other in this comment if that ends up causing you any trouble :-)